product/previewing-and-sharing.mdx

```diff
@@ -12,47 +12,36 @@ icon: "share" iconType: "solid"
+Makeswift allows you to preview and share different versions of your site for enhanced collaboration and feedback using [Site Previews](#site-previews) and [Share Links](#share-links).
 
-Share Links are powered by Site Previews, which means anyone with these links can view the shared version of your site directly on your site's domain itself. This means that site previews can be viewed on any device, including phones and tablets.
-
-You can generate Share Links and/or Site Previews for any of the following:
+You can preview different versions of your site for any of the following:
 
 - The current draft state of your site in the Visual Builder
 - A particular scheduled publish
 - A specific previous publish of your site
 
-## Privacy
+## Site Previews
 
-Shareable links are site-wide previews. This means that a recipient of a share link will be able to view the entire site as it was in the shared version.
+Site Previews allow you to preview different versions of your site directly on your domain without generating a shareable link. This is useful when you want to review changes privately on your own device before deciding whether to share them with others via [Share Links](#share-links).
 
-All share links expire after 5 days. After expiration, visiting the share link will result in an error, and users will no longer be able to see the shared version of the site.
+### Previewing draft state
 
-By default, share links can only be accessed by other members of your Makeswift workspace. To create a public share link, see the [Public Share Links](#public-share-links) section.
+There are two ways to preview the draft state (the current state of your site in the Visual Builder):
 
-## Sharing
+- Via the **View page** icon in the Visual Builder header
+- Via the three dots menu in the Visual Builder header
 
-### Sharing draft state
+To preview via the **View page** icon, click the open icon to the right of the address bar in the Visual Builder header. This opens your site in a new tab and shows the current draft state.
 
-To share the draft state (the current state of your site in the Visual Builder), click on the **Share** button in the top right corner of the Visual Builder header.
-This will open a dialog that has a new shareable link generated for you. You can copy this link and share it with others.
+<Frame>![View page icon](/images/sharing/direct-preview-1.png)</Frame>
 
-<Frame>
-  <video
-    autoPlay
-    muted
-    loop
-    playsInline
-    controls
-    title="Sharing draft state"
-    className="w-full aspect-video"
-    src="/images/sharing/draft-share.mp4"
-  ></video>
-</Frame>
+To preview via the three dots menu, click the three dots in the top right corner of the Visual Builder header. From the menu, select **View page**.
```

Comment:
What do you think about using the name "action menu" instead of "three dots menu" across the docs?